### PR TITLE
add migrations

### PR DESCRIPTION
**Pull request by mixcas**
use version 3 wish resets catalogs, countdown and calendar

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+import { persistStore, persistCombineReducers, createMigrate } from 'redux-persist';
 import storage from 'redux-persist/es/storage'; // default: localStorage if web, AsyncStorage if react-native
 import thunk from 'redux-thunk';
 import reducers from '../reducers';
+import { migrations } from './migrations';
 
 // Redux Persist config
 const config = {
   key: 'root',
+  version: 3,
```
Comment (varffs) [inline src/store/index.js:11]: should this match the specific version of the app? or just this different migration version as another versioning?

Comment (varffs) [reply src/store/index.js:11]: I guess that would require a migration config for each patch version which would be pretty pointless

File: src/store/migrations.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      calendar: undefined,
+    }; // reset all state, except version
+  },
+  3: (state) => {
+    return {
+      ...state,
+      catalogos: undefined,
+      countdown: undefined,
+      calendar: undefined,
+    }; // reset all state, except version
```
Comment (varffs) [inline src/store/migrations.js:21]: why are `2` and `3` the same?

**Review by varffs — COMMENTED** [review]
Question about the config values

File: src/store/index.js (hunk 1 above)
Comment (mixcas) [reply src/store/index.js:11]: This version is just related to the persist

File: src/store/migrations.js (hunk 2 above)
Comment (mixcas) [reply src/store/migrations.js:21]: Cuz we made the tests with v2, and i added v3 just in case ;3

Comment (varffs) [reply src/store/migrations.js:21]: so should we delete 3 and just set 2?

Comment (mixcas) [reply src/store/migrations.js:21]: idk, don't want to risk it for now.

**Pull request merged**